sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1728,6 +1728,14 @@ class DataSourceV2SQLSuiteV1Filter extends DataSourceV2SQLSuite with AlterTableT
     }
   }
 
+  test("REPAIR TABLE omit MSCK") {
```

Comment:
can we move the test to `MsckRepairTableParserSuite`? And probably rename the test suite. And also `MsckRepairTableSuiteBase` and sub classes.

Reply:
oh this is not parser test. But I think a parser change only needs parser test, no end-to-end test.

Reply:
Sounds good, I can remove this one. I agree that e2e tests are not necessary for a parser change.